src/Icon.tsx

```diff
@@ -24,7 +24,7 @@ export { IFaIconProps, FaIcon, FaIconLibrary };
  */
 
 interface IIcon extends BPIconProps {}
-const Icon: React.FunctionComponent<IFaIconProps> = props => {
+const Icon: React.FunctionComponent<BPIconProps> = (props: BPIconProps) => {
```

Comment:
The Blueprint only accepts their `IIconProps`.  Even though our sub-interface makes no changes to the interface, it is still not recognized as a valid interface by the Blueprint button, so I replaced our sub-interface with Blueprint's `IIconProps`.